server/lib/nl/common/constants.py

```diff
@@ -842,6 +842,9 @@
 # Max number of answer places
 MAX_ANSWER_PLACES = 10
 
+# This is used in TOOLFORMER_TABLE mode.  Set to allow "counties in US".
+ABSOLUTE_MAX_PLACES_FOR_TABLES = 3250
```

Comment:
why not making this a bit larger to allow any new county?

Reply:
I think the real number is more like 3143 or something? :)